static/src/stylesheets/module/preferences/_notifications.scss

```diff
@@ -1,20 +1,239 @@
-.notifications-subscribe-link--has-subscriptions {
+$live-notifications-icon-width: 36px;
+$live-notifications-icon-width: 32px;
+$notifications-button-height: 28px;
+
+@mixin notifcations-button-style($foreground: $neutral-2, $background: #ffffff) {
+    .notifications__button {
+        color: $foreground;
+        border-color: $foreground;
+        border-style: solid;
+
+        .inline-icon {
+            fill: $foreground
+        }
+    }
+
+    .notifications__button--subscribed {
+        color: $background;
+        background-color: $foreground;
+        .inline-icon {
+            fill: $background;
+            background-color: $foreground;
+        }
+
+        &:hover {
+            background-color: darken($foreground, 10%);
+            .inline-icon {
+                background-color: darken($foreground, 10%);
+            }
+        }
+    }
+
+    .notifications__button--unsubscribed {
+        color: $foreground;
+        background-color: transparent;
+        .inline-icon {
+            fill: $foreground;
+            background-color: $background;
+        }
+
+        &:hover {
+            background-color: darken($background, 10%);
+            .inline-icon {
+                background-color: darken($background, 10%);
+            }
+
+        }
+    }
+}
+
+@mixin notifications-button-toned($tone, $foreground: $news-main-1, $background: #ffffff) {
+    .tonal--tone-#{$tone} {
+        @include notifcations-button-style($foreground, $background);
+    }
+}
+
+.live-notifications {
+    margin-bottom: $gs-baseline /2;
+    position: relative;
+}
+
+.notifications__button {
+
+    .inline-icon {
+        svg {
+            width: 1.143em;
+            height: .857em;
+        }
+    }
+}
+
+.notifcations-follow-input--solo {
+
+    .submit-input__icon {
+        display: inline-block;
+        margin-right: $gs-gutter / 4;
+    }
+}
+
+.live-notifications__label {
     display: none;
 }
 
-.notifications-subscribe-link__headline {
-    @include fs-header(3);
-    margin: 0;
+@each $state in (subscribed, unsubscribed) {
+    .notifications__button--#{$state} {
+        .live-notifications__label--#{$state} {
+            display: inline;
+        }
+    }
 }
 
-.notifications-link {
-    margin-top: $gs-baseline;
-    margin-bottom: $gs-baseline / 2;
+@include notifcations-button-style();
+@include notifications-button-toned(live, $live-default, $neutral-8);
+@include notifications-button-toned(dead, $live-default, $neutral-8);
+
+.live-notifications-denied {
+    position: fixed;
+    display: block;
+    top: .625rem;
+    left: 5.275rem;
+    width: 26.825rem;
+    height: 4.7625rem;
+    background: fade-out($live-default, 1 - .95);
+    transition: background-color .25s ease;
+    filter: alpha(opacify=50);
+    -moz-opacity: .5;
+    -khtml-opacity: .5;
+    z-index: 15000;
```

Comment:
Should add a zindex value here to avoid zindex chaos https://github.com/guardian/frontend/blob/notifications-ui-redesign/static/src/stylesheets/base/_zindex.scss